pkg/apis/serving/v1/revision_types.go

```diff
@@ -64,6 +64,7 @@ var (
 // RevisionTemplateSpec describes the data a revision should have when created from a template.
 // Based on: https://github.com/kubernetes/api/blob/e771f807/core/v1/types.go#L3179-L3190
 type RevisionTemplateSpec struct {
+	// +kubebuilder:pruning:PreserveUnknownFields
```

Comment:
Tbh, this was kind of a stopgap solution because stuff has been failing without it. We currently print out `deletionTimestamp: null` to every nested ObjectMeta and just the schema would cause this to drop, so I thought the problem might be around that. Hence I added this for compatiblity.